hw/ip_templates/rv_plic/fpv/tb/waivers.tcl

```diff
@@ -0,0 +1,35 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# This assertion states that if the input enable (en_i) is not high and onehot input (oh_i) is
+# non-zero then we get an output error (err_o). The precondition for EnableCheck_A is false i.e.
+# !en_i && |oh_i was never true.
```

Comment:
Nit: I'd probably phrase this as "... cannot become true."

That way, it's clearer that we're describing what's possible, rather than what happened when we ran a test :-)